docker-compose.yml

```diff
@@ -1,5 +1,11 @@
-version: '3'
+version: '3.4'
 services:
+  app:
```

Comment:
Tomuhle nerozumím. Vždyť je to knihovna ne samotnatná aplikace. To je tady zbytečně.

Reply:
jo, to tam nemá co děla. Díky.